src/components/project/handlers/step-changed-notification.handler.ts

```diff
@@ -24,6 +24,7 @@ export class ProjectStepChangedNotificationHandler
   async handle(event: ProjectUpdatedEvent) {
     if (
       event.updated.step === event.previous.step ||
+      event.updated.changeset ||
```

Comment:
If the project step is updated in changesets, I don't think we need to send notifications and it will fix https://github.com/SeedCompany/cord-api-v3/issues/2037.

Reply:
Yes, I agree with `ProjectChangesetFinalized` + `ProjectChangesetAfterFinalized`